### PR TITLE
Arrow key event and movement blocks for Game Lab Jr

### PR DESCRIPTION
**Pull request by balderdash**
image

### CODE REVIEW

File: apps/src/gamelab/blocks.js [hunk 1]
```diff
@@ -53,24 +77,138 @@ export default {
...
+      func: 'moveRight',
+      blockText: 'move right',
+      args: [],
+      methodCall: true,
+    });
+
+    createJsWrapperBlock({
+      category: EVENT_CATEGORY,
+      func: 'whenUpArrow',
+      blockText: 'When up arrow presssed',
```
Comment (joshlory) [inline apps/src/gamelab/blocks.js]: Nit: lowercase "when up arrow pressed".

**Comment by joshlory** [comment]
What does this look like on tablet & phone?  The d-pad got hidden in PR https://github.com/code-dot-org/code-dot-org/pull/20630.

**Comment by joshlory** [comment]
One thing I'm realizing is that it's hard to differentiate visually between "When __ arrow pressed" and "While __ arrow pressed".  I wonder if these should be different color blocks?  But if they're different colors that's confusing as well.  +cc @mrjoshida

Not blocking this PR of course.

**Comment by balderdash** [comment]
We could make the `while` blocks look like loops? :woman_shrugging:

**Comment by balderdash** [comment]
Without the d-pad it's completely unusable on a phone.

**Comment by mrjoshida** [comment]
The loop shaped block idea is really interesting, let's see how that looks

**Comment by balderdash** [comment]
Something like this:
image
Or we could use the traditional loop color:
image

**Comment by balderdash** [comment]
Here's a mix of them in the same workspace:
image

They're still not _that_ visually distinct when they're the same color, I'd lean towards using pink:
image

**Comment by balderdash** [comment]
Merging with pink loops, we can always change things up later.

**Pull request merged**